modules/atomic-widgets/elements/atomic-image/atomic-image.php

```diff
@@ -52,7 +52,7 @@ protected static function define_props_schema(): array {
 				->default( [] ),
 
 			'image' => Image_Prop_Type::make()
-				->default_url( Utils::get_placeholder_image_src() )
+				->default_url( Utils::get_placeholder_image_src( 'v4' ) )
```

Comment:
why do we need to use the same function instead of inlining the URL?
kinda weird to add a param that lets this "core" function know about features ("v4")